### PR TITLE
V1.2beta svc oci spec - update oauth

### PR DESCRIPTION
**Pull request by ramaraosrikakulapu**
Update OAuth for EC service enterpriseconnect/service:v1.2beta image

### CODE REVIEW

File: spec/service/config.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#/bin/sh
```
Comment (ayasuda2OO3) [inline spec/service/config.sh]: Beginning v1.1/v1.1beta release. agent will do direct environment parsing. String replacement is not needed neither to this script.

File: spec/service/config.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+ec-config:
+    conf:
+      mod: oauth2
+      apt: ${EC_PORT}
```
Comment (ayasuda2OO3) [inline spec/service/config.yaml]: ```${PARAM}``` will read/replace by the environment variable $PARAM. No update needed

**Comment by ayasuda2OO3** [comment]
PR added to EC-Release/sdk#133

**Review by ayasuda2OO3 — COMMENTED** [review]
Please consider the change.

File: spec/service/config.sh (hunk 1 above)
Comment (ramaraosrikakulapu) [reply spec/service/config.sh]: So no need of config.sh at all?

**Comment by ramaraosrikakulapu** [comment]
@ayasuda2OO3 .. implemented the suggested changes. Can you please review and approve the request, if looks good

File: spec/service/Dockerfile [hunk 3]
```diff
@@ -20,17 +20,14 @@ WORKDIR /root
...
 ARG EC_REV_V1_2BETA=v1.2beta.fukuoka.0
 
-#COPY ./*.yml ./
-#COPY ./agent.sh ./
+COPY ./config.yaml /root/config.yaml
 
 RUN apk add --no-cache git
 RUN apk add --no-cache -X http://dl-cdn.alpinelinux.org/alpine/edge/testing hub
-RUN wget -O ./agent_linux_sys.tar.gz https://raw.githubusercontent.com/Enterprise-connect/sdk/${EC_REV_V1_2BETA}/dist/agent/agent_linux_sys.tar.gz \
+RUN wget -O ./agent_linux_sys.tar.gz https://raw.githubusercontent.com/EC-Release/sdk/${EC_REV_V1_2BETA}/dist/agent/agent_linux_sys.tar.gz \
```
Comment (ayasuda2OO3) [inline spec/service/Dockerfile]: make it more controllable check this out https://github.com/EC-Release/oci/blob/v1_oci_spec/spec/agent/Dockerfile#L29

Comment (ramaraosrikakulapu) [reply spec/service/Dockerfile]: @ayasuda2OO3 . Looks like we have to deploy new one for EC service v1.2beta version. Will work on it.

Comment (ramaraosrikakulapu) [reply spec/service/Dockerfile]: Added script as suggested. But for now using v1.1beta until v1.2beta agent available

**Review by ayasuda2OO3 — APPROVED** [review]

**Pull request merged**